Volume/Views/MainView/HomeList.swift

```diff
@@ -148,6 +142,12 @@ extension HomeList {
         followedArticles: [Article],
         otherArticles: [Article]
     )
+    typealias ResultsPublisher =
+        Publishers.Zip3<
+            Publishers.Map<GraphQLPublisher<GetTrendingArticlesQuery>,[ArticleFields]>,
+            Publishers.Collect<Publishers.Map<GraphQLPublisher<GetArticlesByPublicationIDsQuery>, [ArticleFields]>>,
+            Publishers.Map<GraphQLPublisher<GetArticlesByPublicationIDsQuery>, [ArticleFields]>
```

Comment:
I feel like this could be a little more readable

Reply:
I completely agree